utils/player.py

```diff
@@ -0,0 +1,25 @@
+
+
+class Player():
+
+    def __init__(self, name):
+        self.__name = name
+        self.__curr_pos = 0
+
+    def move_player_by(self, x, board_size):
```

Comment:
all of this logic should be a part of some service class and not the player model.

Reply:
i didn't get the idea of how to use service class here. can you give me some more idea/hint?

Reply:
A model class should contain only the properties, constructor, getters and setters.
Everything else that is there to interact with the class or contains business logic should be done through different classes but not model classes.